gatsby-node.js

```diff
@@ -271,3 +292,12 @@ const buildNode = (localFiles, row, cleanedKey, raw, mapping, createNodeId) => {
 const cleanKey = (key, data) => {
   return key.replace(/ /g, "_");
 };
+
+const upperFirst = (str) => {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+};
+
+// ensure our type name is valid - remove non a-z chars and make the first letter uppercase 
+const cleanTypeName = (typeName) => {
+  return upperFirst(typeName.replace(/[^a-z]/ig, ''));
```

Comment:
I believe Gatsby takes care of the CamelCasing. Do we need to bother with upperFirst? One concern I have with automatically cleaning is that if we change what they input then it might be confusing when they go to query and it is different. Is it worth putting a warning that we changed it and recommend changing their value in gatsby-config to match?

Reply:
Hey, I'll review this to see if the upperFirst and cleaning can be removed. I believe though as I was developing these fixes that Gatsby was complaining about the formatting not being correct for the names, so I had to do the cleanup. 

I like your idea about warning the user if we change it - that way we can ensure the build still succeeds, but encourage them to create a compliant `typeName` for each table too.

Reply:
👍